### PR TITLE
Fix a problem that frame loop doesn't terminate

### PR DESCRIPTION
**Pull request by RenfengLiu**

### CODE REVIEW

File: gapis/api/vulkan/replay.go [hunk 1]
```diff
@@ -944,6 +944,17 @@ func (a API) Replay(
 		return numInitialCmdWithoutOpt, nil
 	}
 
+	frameLoopEndCmdID := func(cmds []api.Cmd) api.CmdID {
+		lastCmdID := api.CmdID(0)
```
Comment (purvisa-at-google-com) [inline gapis/api/vulkan/replay.go]: is 0 the right default value here? DO we want len(cmds) -1 ?

Comment (RenfengLiu) [reply gapis/api/vulkan/replay.go]: 0 is correct here, as we want to update it to the last terminated cmd. If non is terminated,  and we set to len(cmds) -1, then it is incorrect.

Comment (purvisa-at-google-com) [reply gapis/api/vulkan/replay.go]: A terminated command is one that completed? Or one that was aborted half way through?

Comment (RenfengLiu) [reply gapis/api/vulkan/replay.go]: A terminated command is one that completed. See gapis/api/cmd.go:69

Comment (purvisa-at-google-com) [reply gapis/api/vulkan/replay.go]: OK, logic makes sense to me now! "Terminated" is an unfortunately ambiguous name though :(

**Pull request merged**